manifest.php

```diff
@@ -32,8 +32,31 @@
     'author' => 'Open Assessment Technologies SA',
     'managementRole' => 'http://www.tao.lu/Ontologies/TAOTest.rdf#TaoQtiTestPreviewerRole',
     'acl' => [
-        ['grant', 'http://www.tao.lu/Ontologies/TAOTest.rdf#TaoQtiTestPreviewerRole', ['ext' => 'taoQtiTestPreviewer']],
-        ['grant', ItemAuthorRole::INSTANCE_URI, ['ext' => 'taoQtiTestPreviewer', 'mod' => 'Previewer']],
+        [
+            AccessRule::GRANT,
+            'http://www.tao.lu/Ontologies/TAOTest.rdf#TaoQtiTestPreviewerRole',
+            ['ext' => 'taoQtiTestPreviewer'],
+        ],
+        [
+            AccessRule::GRANT,
+            TaoItemsRoles::ITEM_AUTHOR,
+            ['ext' => 'taoQtiTestPreviewer', 'mod' => 'Previewer'],
+        ],
```

Comment:
Just to understand. Why those 2 were not added to the migrations?

Reply:
These roles were here before - I refactored a bit CS and replaced the deprecated constant with a suitable one 🙂